content/07.pathogenesis.md

```diff
@@ -46,8 +46,12 @@ One severe example includes reports of cytokine storm-like responses in patients
 #### Reproduction Number and Dynamics of Transmission
 
 Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
-The basic reproduction number, R~0~, is the expected number of new infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
-The effective reproduction number, R~t~, describes how the reproduction number may change over time, and is used to quantify deviations in R from R~0~, for example as some fraction of the population becomes infected, or as interventions are put into place.
+The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@ISBN:9780691116174].
+The basic reproduction number, R~0~, is the expected number of new (secondary) infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
+R~0~ is a unitless number which in mechanistic models is a combination of parameters principally related to the transmission rate (rate of infection transmitting interactions for an infectious individual) and the infectious period [@DOI:10.1126/science.abb5659][@ISBN:9780691116174].
+When R~0~ is greater than 1, there is potential for an epidemic to grow.
```

Comment:
```suggestion
A pathogen can invade a susceptible population only if R~0~ > 1 [@ISBN:9780691116174; @ISBN:9780199209996].
```
I believe this is what I'm supposed to do per @yemarshall request, with the assumption that @alavendelm will likely edit in the course of handling #313 !